### DISCUSSION TITLE
SEO Problems

### DISCUSSION THREAD

**Discussion by Locis1** [orestbida/cookieconsent#684] [Q&A]
Hey, I'm a bit confused. When I use the consent tool, my accessibility drops extremely. Is that normal or an issue with my project? 😊

![image](https://github.com/orestbida/cookieconsent/assets/139056715/2c4f4070-ba78-4373-a6a7-1a239c7c46ad)
![image](https://github.com/orestbida/cookieconsent/assets/139056715/2ec0db3e-826b-4c3a-bbc6-23d6dfc1d96d)

**Comment by orestbida** [comment]
I checked locally and I found no issues. You can also check the [playground page](https://playground.cookieconsent.orestbida.com/).

This is probably an unlucky combination specific to your website:

0. dark sphere background
1. big text
2. cookieconsent

Dev tools is not happy if all 3 of them exist; it fails to distinguish what is background and what is foreground, and it's unable to tell where exactly is the issue (since there really isn't one).

Don't how this can be solved without modifying the website.

By the way, I noticed you are loading a huge image (~11MB) and there is no gzip/brotli compression enabled for javascript/css assets.

**Comment by Locis1** [comment]
Yeah, it's a very bad combination and you are right. 

I tested it now without the Ball and the big text in the background and it works fine. 

Thank u for your help !

And yeah I know that that im loading a big image, I wanted to optimise my Website step by step and I was stuck with the Accessibility 😇


![image](https://github.com/orestbida/cookieconsent/assets/139056715/3e864fcb-cb81-40f4-83cb-def5e2c5f708)
